docs/bylaws.md

```diff
@@ -0,0 +1,51 @@
+OmnICU-SC Bylaws
+================
+
+OmnICU-SC abides by the bylaws listed in this document.
+
+## Decision Making
+
+### Technical Decisions
+
+OmnICU-TC maintains a collection of explainers (design documents) that are used as a basis for technical decisions.  All explainers that have been approved by OmnICU-SC live as Markdown files on GitHub.  *Designs are not official unless they are in an explainer checked-in to GitHub.*
```

Comment:
Could there be a separate document from these bylaws that gives tips / howto for contributing?  And maybe it would start off only containing a link to a good tool for making the SVG diagrams for explainers, and how people view such diagrams in an explainer PR.

Reply:
That's a good idea.  I'll do it in a follow-up PR.

Reply:
Done: #21